pandas/core/generic.py

```diff
@@ -6881,30 +6881,41 @@ def interpolate(
         inplace = validate_bool_kwarg(inplace, "inplace")
 
         axis = self._get_axis_number(axis)
+        index = self._get_axis(axis)
+
+        if isinstance(self.index, MultiIndex) and method != "linear":
+            raise ValueError(
+                "Only `method=linear` interpolation is supported on MultiIndexes."
+            )
+
+        if method in ["backfill", "bfill", "pad", "ffill"]:
```

Comment:
can you add a comment here that for these methods, limit_direction and limit_area are being ignored and include a link to https://github.com/pandas-dev/pandas/issues/26796

Reply:
done